Makefile

```diff
@@ -137,15 +152,27 @@ build:
 
 # Run a command in the docker image running kind. Command passed as "TARGET" env.
 kind-run:
-	docker exec -e KUBECONFIG=/etc/kubernetes/admin.conf  \
+ifeq ($(ONE_NAMESPACE), 1)
+	docker exec -e KUBECONFIG=/etc/kubernetes/admin.conf -e ONE_NAMESPACE=1 \
+		${KIND_CLUSTER}-control-plane \
+		bash -c "cd ${GOPATH}/src/istio.io/installer; make ${TARGET}"
+else
+	docker exec -e KUBECONFIG=/etc/kubernetes/admin.conf \
 		${KIND_CLUSTER}-control-plane \
 		bash -c "cd ${GOPATH}/src/istio.io/installer; make ${TARGET}"
+endif
 
 # Runs the test in docker. Will exec into KIND and run "make $TEST_TARGET" (default: run-all-tests)
 docker-run-test:
-	docker exec -e KUBECONFIG=/etc/kubernetes/admin.conf  \
+ifeq ($(ONE_NAMESPACE), 1)
```

Comment:
I wonder if we can just pass down all env variables so we don't need special cases like this? Not needed for this PR but something to keep in mind for the future.

nit: can we just do ONE NAMESPACE=$(ONE NAMESPACE) instead of if statement?

Reply:
nice catch.